components/content/zh/page-sections.md

```diff
@@ -0,0 +1,3 @@
+---
+edit: '编辑'
```

Comment:
Oh, I added two things to this file last night. Could you give that a look. Thanks!